Dockerfile

```diff
@@ -49,8 +50,7 @@ cp -r /src/gurobi/linux64/lib /opt/gurobi/linux64/lib
 cp -r /src/gurobi/linux64/bin /opt/gurobi/linux64/bin
 rm -rf /opt/gurobi/linux64/{lib, bin}/python*
 
-cd /src/gurobi/linux64
-python setup.py install
+pip install "gurobipy==${GUROBI_VERSION}"
```

Comment:
Any reason this can't be tracked by Poetry? Possibly in an optional dependency group to avoid a hard dependency and then do a `poetry export --with gurobi` below.

https://python-poetry.org/docs/managing-dependencies/#optional-groups

It'll let us keep track of transitive dependencies and make sure everything is compatible and in non-container environments you can also do a `pip install adaptive-scheduler[gurobi]`

Reply:
Ive added a gurobi optional group, but for some reason just a plain `poetry install` also installs gurobipy. I don't think this will cause issues for anyone, but I can't figure out why its happening - it looks to me like I have it set to optional correctly according to the docs.